src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourceBuiltArtifactsTests.cs

```diff
@@ -48,17 +49,19 @@ public void VerifyVersionFile()
             }
 
             // Verify the SDK version
+            if (!string.IsNullOrWhiteSpace(Config.SdkTarballPath))
```

Comment:
This will cause the test case to pass if the SDK tarball isn't provided. That's not valid. It should fail. We can't accurately validate the version file without having the SDK tarball.

Reply:
Good call. I think I made this change for local testing and then accidentally committed it. The latest commit reverts the change.